Content.Server/GameObjects/Components/Observer/GhostComponent.cs

```diff
@@ -94,43 +94,48 @@ public override void HandleNetworkMessage(ComponentMessage message, INetChannel
                         Owner.EntityManager.EventBus.RaiseEvent(EventSource.Local, new GhostReturnMessage(mind.Mind));
                     }
                     break;
-                case GhostWarpRequestMessage warp:
+                case GhostWarpToNameRequestMessage warp:
                 {
                     if (session?.AttachedEntity != Owner)
                     {
                         break;
                     }
 
-                    if (warp.PlayerTarget != default)
+                    foreach (var warpPoint in FindWaypoints())
                     {
-                        if (!Owner.EntityManager.TryGetEntity(warp.PlayerTarget, out var entity))
+                        if (warp.WarpName == warpPoint.Location)
                         {
+                            Owner.Transform.Coordinates = warpPoint.Owner.Transform.Coordinates;
                             break;
                         }
```

Comment:
Maybe log a warning here if a user tries to warp to a nonexistant warp location?

Reply:
Done in https://github.com/space-wizards/space-station-14/pull/3310/commits/141de1126f31e66cc5db5d954ab65aadc3fbf794